src/main/java/frc/robot/subsystems/AmpBar.java

```diff
@@ -1,84 +1,89 @@
 package frc.robot.subsystems;
-import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
 import edu.wpi.first.math.controller.PIDController;
-import edu.wpi.first.wpilibj.DutyCycleEncoder;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
-import com.ctre.phoenix.motorcontrol.NeutralMode;
-
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkBase.IdleMode;
 import frc.robot.Constants;
 import frc.robot.Robot;
 
 public class AmpBar {
-    enum AmpBarStates {
+    public enum AmpBarStates {
         IN,
-        OUT
+        SHOOTING,
+        OUT,
     }
-  
-    PIDController pivotController = new PIDController(1.5, 0, 0);
-    private AmpBarStates state = AmpBarStates.OUT;
-    private final WPI_TalonSRX leftMotor = new WPI_TalonSRX(40);
-    private final WPI_TalonSRX rightMotor = new WPI_TalonSRX(45);
-    private DutyCycleEncoder pivotEncoder = new DutyCycleEncoder(9);
+
+    private AmpBarStates state = AmpBarStates.IN;
+    private final CANSparkMax rightMotor = new CANSparkMax(31, MotorType.kBrushless);
+    private final CANSparkMax leftMotor = new CANSparkMax(30, MotorType.kBrushless);
+    private final TalonFX wheelsMotor = new TalonFX(38);
+
+    public RelativeEncoder pivotEncoder = leftMotor.getEncoder();
     double pivotMotorSetpoint = Constants.AmpBar.IN;
     String stateString = "";
     Robot robot = null;
-    
+
+    PIDController controller = new PIDController(1, 0, 0); // TODO: tune PID
```

Comment:
probably shouldn't be merged without PID tuning? up to you

Reply:
I don't think we should merge until testing tomorrow once we get the belt on the last roller